installation/resources/kyma/installer-overrides-api-gateway.yaml

```diff
@@ -8,5 +8,5 @@ metadata:
     component: api-gateway
     kyma-project.io/installation: ""
 data:
-  tests.env.gatewayName: "compass-istio-gateway"
-  tests.env.gatewayNamespace: "compass-system"
+  tests.env.gatewayName: "kyma-gateway"
```

Comment:
This are the same values as the defaults from kyma we can remove the whole file.

Reply:
The file is removed